entrypoint.sh

```diff
@@ -11,7 +11,7 @@ set -o errexit
 SOURCE_DIRECTORY=${GITHUB_WORKSPACE}/$INPUT_SOURCE
 DESTINATION_DIRECTORY=${GITHUB_WORKSPACE}/$INPUT_DESTINATION
 PAGES_GEM_HOME=$BUNDLE_APP_CONFIG
-GITHUB_PAGES=$PAGES_GEM_HOME/bin/github-pages
+GITHUB_PAGES_BIN=$PAGES_GEM_HOME/bin/github-pages
```

Comment:
We've more recently added an exported environment variable `GITHUB_PAGES=true` for our `actions/deploy-pages` action, so I thought it would be best to avoid confusion here.